### PR TITLE
Change workspace icon how to page

### PR DESCRIPTION
**Pull request by gdeflaux**
As promised, here is the how to page to change the workspace icons based on our discussion. Also fixed a typo in the README.

### CODE REVIEW

File: content/en/docs/HowTo/change-workspace-icons.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+You can browse the icon font using the [Gnome Character Map](https://wiki.gnome.org/action/show/Apps/Gucharmap?action=show&redirect=Gucharmap) application (`gucharmap` package):
+
+1. Open Gnome Character Map.
+2. Select your icon font in the top left combo.
+3. In the _script_ pane select _Common_.
+4. Scroll down the characters pane until you find the icons.
+5. Double click on the icon you looking to use and press the _Copy_ button (bottom right).
+
+**Tip:** the icons are order alphabetically.
```
Comment (beyera) [inline content/en/docs/HowTo/change-workspace-icons.md]:
```diff
- **Tip:** the icons are order alphabetically.
+ **Tip:** the icons are ordered alphabetically.
```

File: content/en/docs/HowTo/change-workspace-icons.md [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+Icons are actually characters from fonts installed by Regolith looks. Each look comes with a specific icon font. The icon font is defined in the `typeface_bar` variable in the `typeface` config file. The default look Cahuella uses the _Material Design Icons_ font (see file `/etc/regolith/styles/cahuella/typeface`).
+
+You can browse the icon font using the [Gnome Character Map](https://wiki.gnome.org/action/show/Apps/Gucharmap?action=show&redirect=Gucharmap) application (`gucharmap` package):
+
+1. Open Gnome Character Map.
+2. Select your icon font in the top left combo.
+3. In the _script_ pane select _Common_.
+4. Scroll down the characters pane until you find the icons.
+5. Double click on the icon you looking to use and press the _Copy_ button (bottom right).
```
Comment (beyera) [inline content/en/docs/HowTo/change-workspace-icons.md]: "you are looking to use"
```diff
- 5. Double click on the icon you looking to use and press the _Copy_ button (bottom right).
+ 5. Double click on the icon you are looking to use and press the _Copy_ button (bottom right).
```

**Comment by beyera** [comment]
Thank's for adding this doc! I saw the conversation in Slack and know this will be useful for others. 

I suggested a couple of minor readability changes.

**Comment by kgilmer** [comment]
Beautiful!  Thank you @beyera .  DM me a postal address if you'd like a few Regolith laptop stickers. :smile:

**Comment by beyera** [comment]
@gdeflaux did all the work here and deserves the credit!

**Comment by gdeflaux** [comment]
Thanks guys. @beyera, thanks for proof reading this.

**Comment by kgilmer** [comment]
Oh, whoops I didn't release there were two people on this PR :man_facepalming:   Thanks @gdeflaux!  I've applied @beyera 's suggestions manual and pushed an update.

**Pull request merged**